MBBSEmu.Tests/ExportedModules/Majorbbs/itoa_Tests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class itoa_Tests : ExportedModuleTestBase
+    {
+        private const int ITOA_ORDINAL = 366;
+
+        [Theory]
+        [InlineData(4, 10, "4")]
+        [InlineData(500, 10, "500")]
+        [InlineData(8500, 8, "20464")]
+        [InlineData(8500, 16, "2134")]
+        [InlineData(5, 2, "101")]
+        [InlineData(1000, 16, "3e8")]
+        [InlineData(1000, 2, "1111101000")]
+        public void itoa_Test(ushort intConvert, ushort baseInt, string expected)
+        {
+            //Reset State
+            Reset();
+
+            //Set Argument Values to be Passed In
+            var destinationStringPointer = mbbsEmuMemoryCore.AllocateVariable("STRDST", (ushort)(expected.Length + 1));
+            mbbsEmuMemoryCore.SetArray("STRDST", Encoding.ASCII.GetBytes("STRDST"));
+
+            //Execute Test
+            ExecuteApiTest(HostProcess.ExportedModules.Majorbbs.Segment, ITOA_ORDINAL,
+                new List<ushort>
+                {
+                    intConvert,
+                    destinationStringPointer.Offset,
+                    destinationStringPointer.Segment,
+                    baseInt
+                });
+
+            //Verify Results
+            Assert.Equal(expected, Encoding.ASCII.GetString(mbbsEmuMemoryCore.GetString("STRDST", true)));
```

Comment:
we'll want to assert on `DX:AX` as well

Reply:
DX:AX is 0 because we don't return anything right?

Reply:
itoa returns a `char *`, so this should be `Assert.NotEqual(0)`

The ordinal needs to return `dest`